pkgs/applications/blockchains/exodus/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, lib, fetchurl, unzip, glib, systemd, nss, nspr, gtk3-x11, pango,
 atk, cairo, gdk-pixbuf, xorg, xorg_sys_opengl, util-linux, alsaLib, dbus, at-spi2-atk,
-cups, vivaldi-ffmpeg-codecs, libpulseaudio, at-spi2-core }:
+cups, vivaldi-ffmpeg-codecs, libpulseaudio, at-spi2-core, libxkbcommon, mesa_noglu }:
```

Comment:
mesa_noglu does not exist.

Reply:
![image](https://user-images.githubusercontent.com/9866621/104200517-ad69a280-5420-11eb-962c-af88ba595c58.png)

https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+sort%3Aupdated-desc+mesa_noglu+is%3Aclosed

https://github.com/NixOS/nixpkgs/pull/40438/files#diff-119b59a5c3a9310171e85e5ff6c05b1135904b4accc30d92cf3b9b98dedd3f41R495

:shrug: I don't usually package graphical programs for nix so some guidance would be appreciated.
Is it not building for you? Are you using your nixos machine?

Reply:
It is not evaling. It makes no difference if I use nixos or not.

Reply:
The package is only mesa and mesa_noglu is an alias which is ignored for ofborg. https://github.com/NixOS/nixpkgs/blob/66c01cf6f104997bfe25499ecc558f4dcc886a8b/pkgs/top-level/aliases.nix#L338

Reply:
So swapping it to just `mesa` works?

Reply:
Should.

Reply:
Resolved, thanks